docs/UserGuide.md

```diff
@@ -154,6 +154,12 @@ Examples:
 
 * `find Betsy` followed by `delete 1` deletes the 1st guest in the results of the `find` command.
 
+### Marking room clean status: `markRoomUnclean`
```

Comment:
Should this be `Marking all rooms as unclean` or similar?

Reply:
Yes, I will take your suggestion.